axelrod/strategy_transformers.py

```diff
@@ -37,6 +39,8 @@ def __call__(self, player, opponent, action)
         Any keyword arguments to pass to the wrapper
     name_prefix: string, "Transformed "
         A string to prepend to the strategy and class name
+    reclassifier: function,
+        A function to use to update the class's classifier.
```

Comment:
Minor quibbles:

1. Somebody new to this might reasonably ask "What class?"
2. "to use" is redundant

How about: "A function which will update the classifier of the strategy being transformed"?